pkg/pkgmgr/rpm.go

```diff
@@ -107,19 +108,26 @@ func isLessThanRPMVersion(v1, v2 string) bool {
 }
 
 // Map the target image OSType & OSVersion to an appropriate tooling image.
-func getRPMImageName(manifest *unversioned.UpdateManifest) string {
-	// Standardize on mariner as tooling image base as redhat/ubi does not provide
-	// static busybox binary
-	image := "mcr.microsoft.com/cbl-mariner/base/core"
-	version := "2.0"
-	if manifest != nil && manifest.Metadata.OS.Type == "cbl-mariner" {
-		// Use appropriate version of cbl-mariner image if available
-		vers := strings.Split(manifest.Metadata.OS.Version, ".")
-		if len(vers) < 2 {
-			vers = append(vers, "0")
-		}
-		version = fmt.Sprintf("%s.%s", vers[0], vers[1])
+func getRPMImageName(manifest *unversioned.UpdateManifest, osType string, osVersion string) string {
+	var image, version string
+
+	if osType == "azurelinux" {
+		image = "mcr.microsoft.com/azurelinux/base/core"
```

Comment:
I think we should also update the FAQ tooling image section to reflect this tooling image for azurelinux

Reply:
updated